packages/samples/specs/resource-manager/arm-library/main.tsp

```diff
@@ -17,12 +17,19 @@ using Azure.ResourceManager;
 @OpenAPI.info({
   version: "2021-09-21-preview",
 })
-@useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
 @useLibraryNamespace(Microsoft.InterfacesTest, Microsoft.OperationsTest)
-@armCommonTypesVersion(CommonTypes.Versions.v5)
 @armProviderNamespace
+@versioned(Versions)
 namespace Microsoft.LibraryTest;
 
+/** Contoso API versions */
+enum Versions {
+  /** 2021-10-01-preview version */
+  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
+  @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5)
+  `2021-10-01-preview`,
+}
+
```

Comment:
There was some copy pasting I think:
```suggestion
/** ARM Library API versions */
enum Versions {
  /** 2021-09-21-preview version */
  @useDependency(Azure.ResourceManager.Versions.v1_0_Preview_1)
  @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v5)
  `2021-09-21-preview`,
}

```
I think we can delete the `@OpenAPI.info` decorator as well? (should we warn if that's present with `@versioned`?)

Reply:
yeah should definitely delete it from the samples and yeah maybe but in general for azure we don't want people to use openapi decorators at all